src/stores/UI.ts

```diff
@@ -0,0 +1,30 @@
+import { AsyncStorage } from 'react-native';
+import { types, flow, applySnapshot, onSnapshot } from 'mobx-state-tree';
+import { config } from '~config';
+
+export const UI = types.model('UI', {
```

Comment:
What about `const UI` and at then end `export default UI;`?

Reply:
Named imports/exports promote clarity. In addition, current tooling differs on the correct way to handle default imports/exports. Avoiding them all together can help avoid tooling bugs and conflicts.

https://palantir.github.io/tslint/rules/no-default-export/

Reply:
Okay this one is related to components imports then, see my comment below